src/Datasource/EntityTrait.php

```diff
@@ -466,18 +466,13 @@ public function has(array|string $field): bool
     public function isEmpty(string $field): bool
     {
         $value = $this->get($field);
-        if (
-            $value === null ||
-            (
-                is_array($value) &&
-                empty($value) ||
-                $value === ''
-            )
-        ) {
-            return true;
-        }
 
-        return false;
+        return $value === null ||
+        (
+            is_array($value) &&
+            empty($value) ||
+            $value === ''
+        );
```

Comment:
Neither of these is great. Think this is a place where multiple return paths is easier.